frontend/src/pages/PickupGem/FoundGem.vue

```diff
@@ -0,0 +1,94 @@
+<template>
+  <div class="bg-container">
+    <video autoplay muted :poster="getPoster" id="gem-video">
+      <source :src="getVideo" type="video/mp4">
+    </video>
+    <div id="found-gem">
+      <h1>You found a gem! {{ gemPackage.color }}</h1>
+    </div>
+    <div id="view-button">
+      <van-button @click="$emit('nextStage')">View contents</van-button>
+    </div>
+  </div>
+</template>
+<script lang="ts">
+import { defineComponent, PropType } from "vue";
+import { Button } from "vant";
+import { GemInfo, color } from "@/interfaces";
+
+export default defineComponent({
+  props: {
+    gemPackage: {
+      type: Object as PropType<GemInfo>,
+      required: true,
+    }
+  },
+  components: { 
+    "van-button": Button,
+  },
+  computed: {
+    getPoster() {
+      // is this bad code, should i use switch instead :-)
+      if (Object.values(color).includes(this.gemPackage.color as color)) {
+        return require('@/assets/images/gem-' + this.gemPackage.color +'-found.png');
+      }
+      return require('@/assets/images/gem-purple-found.png');
```

Comment:
looks good! For line 35, we don't expect this case to occur right? Maybe we can add an assert that `Object.values(color).includes(this.gemPackage.color as color)` then just `return require('@/assets/images/gem-' + this.gemPackage.color +'-found.png');`?

Reply:
Yea, I wrote it as a fallback... okay, sounds good!

Reply:
> Also, for the popup drawer, I'm just wondering if it's necessary for us to collapse it to begin with?

I think it's gonna be pretty big when fully expanded?